fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/ACosExpNode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
+
+import java.util.Stack;
+
+/**
+ * The ACOS SQL function
+ */
+public class ACosExpNode implements ExpNode {
+    private final ExpNode arg;
+
+    /**
+     * Public constructor
+     * @param refs
+     */
+    public ACosExpNode(ExpNode arg) {
+        this.arg = arg;
+    }
+
+    @Override
+    public <T> T visit(ExpNodeVisitor<T> visitor) {
+        return visitor.acos(arg.visit(visitor));
+    }
+
+    @Override
+    public int precedence() {
+        return 5;
```

Comment:
Perhaps have a common enum/utility method that gets precedence int based on passed in class; having all the relative numbers in one place may be easier to see the relative precedence of the classes.

Reply:
I think the precedence is an attribute that belongs to each class, so I'd like to keep it internal. However, I do think having some sort of documentation somewhere containing each precedence level and the nodes in that level is a good idea.

Reply:
Added a README.md